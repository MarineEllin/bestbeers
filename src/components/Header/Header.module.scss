.headerContainer {
  background-color: var(--primary);
  box-shadow: var(--box-shadow);
  .headerContent {
    display: flex;
    flex-direction: row;
    justify-content: space-evenly;
    align-items: center;
    padding: 20px;
    .title {
      font-size: x-large;
    }
    .logo {
      width: 250px;
      height: 250px;
    }
  }
}
