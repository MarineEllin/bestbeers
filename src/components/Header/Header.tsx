import styles from "./Header.module.scss";

function Header() {
  return (
    <div className={styles.headerContainer}>
      <div className={styles.headerContent}>
        <img src="logo.png" alt="logo" className={styles.logo} />
      </div>
    </div>
  );
}

export default Header;
