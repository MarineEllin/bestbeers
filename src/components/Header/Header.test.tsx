import renderer from "react-test-renderer";
import Header from "./Header";

test("Write a test for Header with snapshot", () => {
  const component = renderer.create(<Header />);

  let tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
