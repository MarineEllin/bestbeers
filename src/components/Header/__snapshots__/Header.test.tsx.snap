// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Write a test for Header with snapshot 1`] = `
<div
  className="headerContainer"
>
  <div
    className="headerContent"
  >
    <img
      alt="logo"
      className="logo"
      src="logo.png"
    />
  </div>
</div>
`;
